tests/unit_functional

```diff
@@ -12,20 +12,21 @@ set +a
 
 printf "\nRunning PHPUnit functional tests\n"
 
-echo "Preparing the database for functional tests..."
-exec 3>&1
-exec 1> /dev/null
-pg_restore -h $GIGADB_HOST -U $GIGADB_USER -d $GIGADB_DB --clean --no-owner -v /var/www/sql/gigadb_testdata.pgdmp || true
-exec 1>&3
+#echo "Preparing the database for functional tests..."
```

Comment:
I think the changes in this file have been included in this pull request by mistake?